pkg/cmd/roachtest/tests/asyncpg.go

```diff
@@ -45,13 +51,16 @@ func registerAsyncpg(r registry.Registry) {
 
 		t.Status("cloning asyncpg and installing prerequisites")
 
+		t.L().Printf("Latest asyncpg release is %s.", asyncpgSupportedTag)
```

Comment:
the "latest" line is not accurate. it should try to fetch the latest tag from the repo. for example,  see the hibernate test:
```
		latestTag, err := repeatGetLatestTag(
			ctx, t, "hibernate", "hibernate-orm", hibernateReleaseTagRegex,
		)
```